docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -0,0 +1,554 @@
+    "    url=\"https://s3.us-east-1.wasabisys.com/iguazio/project-archive/project-archive.zip\",\n",
+    "    clone=True\n",
+    ")\n",
+    "\n",
+    "archive_project.set_source(source=None, pull_at_runtime=True)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d053a867",
+   "metadata": {},
+   "source": [
+    "Then, you can use `set_function` with `with_repo=True` create the function.\n",
+    "> Note: In this case, there is a Python file called `job_func.py` in the root directory of the archive:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fa58f11",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "archive_fn = git_project.set_function(\n",
+    "    name=\"archive-function\",\n",
+    "    func=\"./job_func.py\",\n",
+    "    kind=\"job\",\n",
+    "    image=\"mlrun/mlrun\",\n",
+    "    handler=\"job_handler\",\n",
+    "    with_repo=True\n",
+    ")\n",
+    "\n",
+    "run = archive_project.run_function(archive_fn)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d59aac4",
+   "metadata": {},
+   "source": [
+    "<a id=\"import-existing-function\"></a>\n",
+    "### Import Existing Function (YAML, Function Marketplace, MLRun DB)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c2c7b6e",
+   "metadata": {},
+   "source": [
+    "If you already have an MLRun function that you would like to import, you can do so from multiple locations such as YAML, function marketplace, and MLRun DB."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c08b6a6a",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "import_project = mlrun.new_project(name=\"import-project\", context=\"./import_project\", overwrite=True)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0cf2b5f2",
+   "metadata": {},
+   "source": [
+    "#### YAML"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1718a30b",
+   "metadata": {},
+   "source": [
+    "MLRun functions can be exported to YAML files via `fn.export()`. These YAML files can then be imported via the following:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "185088f9",
+   "metadata": {},
+   "source": [
+    "```python\n",
+    "yaml_fn = import_project.set_function(name=\"import\", func=\"function.yaml\")\n",
+    "\n",
+    "run = import_project.run_function(yaml_fn)\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6866ab43",
+   "metadata": {},
+   "source": [
+    "#### Function Marketplace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a51e96d",
+   "metadata": {},
+   "source": [
+    "Functions can also be imported from the [MLRun Function Marketplace](https://www.mlrun.org/marketplace) - simply import via the name of the function and the `hub://` prefix:\n",
+    "> Note: By default, the `hub://` prefix will point to the official marketplace, however you can also substitute your own repo to create your own marketplace"
```

Comment:
Note: By default, the `hub://` prefix points to the official marketplace, however you can also substitute your own repo to create your own marketplace.